db/model/roomType.js

```diff
@@ -69,6 +69,9 @@ module.exports = function roomType(seq, dataTypes) {
         assoc.type = RoomType.hasMany(models.Room, {
           foreignKey: 'type',
         });
+        assoc.bots = RoomType.hasMany(models.Bot, {
+          foreignKey: 'roomTypeId',
```

Comment:
That's what I thought too but when I use the "through" tag with hasMany, I get an error saying this isn't allowed